production/nomad/loki/job.nomad.hcl

```diff
@@ -0,0 +1,98 @@
+job "loki" {
+  datacenters = ["dc1"]
+
+  group "loki" {
+    count = 1
+
+    ephemeral_disk {
+      # Used to store index, cache, WAL
+      # Nomad will try to preserve the disk between job updates
+      size   = 1000
+      sticky = true
+    }
+
+    network {
+      port "http" {
+        to     = 3100
+        static = 3100
+      }
+      port "grpc" {}
+    }
+
+    task "loki" {
+      driver = "docker"
+      user   = "nobody"
+
+      config {
+        image = "grafana/loki:2.5.0"
```

Comment:
Q: can we customize the Loki image during `nomad run ..`? . Can you we also make it as `locals.version` instead of hard coding it?

Reply:
It can be done but with different syntax. I will update.